devices/stm32l4x3.yaml

```diff
@@ -11,6 +11,20 @@ RCC:
       USBF:
         name: USBFSEN
         description: USB FS clock enable
+        
+    # SVD incorrectly shifts CRCEN/CRCRST/CRCSMEN  11 bits instead of 12
+    AHB1ENR:
+        _modify:
+            CRCEN:
+                bitOffset: 12
+    AHB1RSTR:
+        _modify:
+            CRCRST:
+                bitOffset: 12
+    AHB1SMENR:
+        _modify:
+            CRCSMEN:
+                bitOffset: 12
```

Comment:
```suggestion
  # SVD incorrectly shifts CRCEN/CRCRST/CRCSMEN  11 bits instead of 12
  AHB1ENR:
    _modify:
      CRCEN:
        bitOffset: 12
  AHB1RSTR:
    _modify:
      CRCRST:
        bitOffset: 12
  AHB1SMENR:
    _modify:
      CRCSMEN:
        bitOffset: 12
```

Reply:
Awesome!  Thanks for the help with the fix